### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by multiphaseCFD**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [x] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [x] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**
Correct typo errors.
**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

**Comment by multiphaseCFD** [comment]
Need to merge the new updates of PR#14.

**Review by AmintorDusko — APPROVED** [review]
Nice! Thank you for that!

File: pennylane_lightning_kokkos/src/algorithms/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-project(lightning_gpu_algorithms LANGUAGES CXX)
+project(lightning_kokkos_algorithms LANGUAGES CXX)
```
[the remark this answers is not in the document]
Comment (multiphaseCFD) [reply pennylane_lightning_kokkos/src/algorithms/CMakeLists.txt:1]: No, it doesn't conflict with anything else.

**Comment by multiphaseCFD** [comment]
Thanks @mlxd and @AmintorDusko for the quick review!

**Pull request merged**